### PR TITLE
Rephrase csaf validation result

### PR DESCRIPTION
**Pull request by JanHoefelmeyer**
Give out a warning if no remote validator is configured.

Clarify that the inbuild validator only validates against the schema.

Solves https://github.com/csaf-poc/csaf_distribution/issues/338

### CODE REVIEW

File: cmd/csaf_checker/reporters.go [hunk 1]
```diff
@@ -49,12 +49,12 @@ func (bc *baseReporter) requirement(domain *Domain) *Requirement {
...
 	if p.validator == nil {
-		req.message(InfoType, "No remote validator configured")
+		req.message(WarnType, "No remote validator configured")
 	}
 	if !p.invalidAdvisories.used() {
 		req.message(InfoType, "No validations performed")
 	} else if len(p.invalidAdvisories) == 0 {
-		req.message(InfoType, "All advisories validated fine.")
+		req.message(InfoType, "All advisories validated fine against the schema.")
 	} else {
```
Comment (tschmidtb51) [inline cmd/csaf_checker/reporters.go:58]: I think we need to extend the logic here:
If a remote validator and one of the presets `basic`, `mandatory`, `extended` or `full` was used, we should say `"All advisories validated fine."`
If a remote validator and none of the presets `basic`, `mandatory`, `extended` or `full` was used or no remote validator was used, we should say `"All advisories validated fine against the schema."`

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
Please see my comment.

**Review by s-l-teichmann — APPROVED** [review]
LGTM

**Review by tschmidtb51 — APPROVED** [review]
LGTM and works

**Pull request merged**